notebooks/00-release-28.0.0/03-lookup-tables-28.ipynb

```diff
@@ -0,0 +1,1129 @@
+    "### Import additional modules\n",
+    "\n",
+    "Run the following cell to import additional modules for this notebook.\n",
+    "\n",
+    "The cell also contains two functions - `postLookup` and `waitForLookup`, wrapping API posts to Druid when working with lookups.\n",
+    "\n",
+    "* You will use the `postLookup` function to call the [lookup configuration API](https://druid.apache.org/docs/latest/api-reference/lookups-api) to create and update lookup tables.\n",
+    "* The `waitForLookup` function will be used to give you feedback on Druid's progress in [distributing](https://druid.apache.org/docs/latest/querying/lookups#configuration-propagation-behavior) the lookup table around the query-serving processes in Druid."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc4c2524-0eba-4bc6-84ed-da3a25aa5fbe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import requests\n",
+    "import time\n",
+    "\n",
+    "def postLookup(definition):\n",
+    "    x = requests.post(druid_host + '/druid/coordinator/v1/lookups/config', json=definition)\n",
+    "\n",
+    "    if \"error\" in x.text:\n",
+    "        raise Exception('Not able to complete the request. \\n\\n'+x.text)\n",
+    "    else:\n",
+    "        print('Successfully submitted the lookup request.')\n",
+    "\n",
+    "def waitForLookup(tier, name, ticsMax):\n",
+    "\n",
+    "    # The default time period between checks of lookup definition changes (druid.manager.lookups.period)\n",
+    "    # is two minutes. The notebook environment reduces this for learning purposes.\n",
+    "    # \n",
+    "    # https://druid.apache.org/docs/latest/configuration/#lookups-dynamic-configuration\n",
+    "\n",
+    "    tics = 0\n",
+    "    ticsWait = 1    \n",
+    "    ticsMax = min(ticsMax,360)\n",
+    "    ticsSpinner = \"/-\\|\"\n",
+    "    \n",
+    "    apicall = druid_host + '/druid/coordinator/v1/lookups/status/'+tier+'/'+name+'?detailed=true'\n",
+    "\n",
+    "    x = requests.get(apicall)\n",
+    "\n",
+    "    while (x.text != '{\"loaded\":true,\"pendingNodes\":[]}' and tics < ticsMax):\n",
+    "        print(x.text + ' ' + ticsSpinner[tics%len(ticsSpinner)] + ' [' + str(ticsMax-tics) + ']   ', end='\\r')\n",
+    "        time.sleep(ticsWait)\n",
+    "        tics += 1\n",
+    "        x = requests.get(apicall) \n",
+    "\n",
+    "    if (tics == ticsMax):\n",
+    "        raise Exception('\\nTimeout waiting for Druid to load the ' + name + ' lookup to ' + tier + 'tier. Run the cell again.')\n",
+    "    else:\n",
+    "        print('\\nSuccess. ' + name + ' lookup in ' + tier + ' tier is fully available.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71dca490-ac78-40b3-b043-16920e379d87",
+   "metadata": {},
+   "source": [
+    "### Initialize the lookups feature\n",
+    "\n",
+    "Before a table can be created in the lookup schema, it must be initialized.\n",
```

Comment:
```suggestion
    "You must initialized the lookups feature before creating a table in the lookup schema.\n",
```
Massive passive. It isn't clear if you need to initialize the table or the feature.